import_streets/src/extract.rs

```diff
@@ -125,24 +119,17 @@ impl OsmExtract {
             return false;
         }
 
-        if highway == "track" && tags.is("bicycle", "no") {
-            return false;
-        }
-
-        #[allow(clippy::collapsible_if)] // better readability
-        if (highway == "footway" || highway == "path" || highway == "steps")
-            && opts.map_config.inferred_sidewalks
-        {
-            if !tags.is_any("bicycle", vec!["designated", "yes", "dismount"]) {
-                return false;
+        // If we're only handling sidewalks tagged on roads, skip a bunch of ways
```

Comment:
I expect this to go away in my next planned step, which'll hopefully be introducing lane types for footpaths (anything that isn't parallel to a "normal road") and shared-use things (foot+cycle shared spaces)